step-templates/liquibase-run-command.json

```diff
@@ -22,7 +22,7 @@
asePassword = (aws rds generate-db-auth-token --hostname $liquibaseServerName --region $region --port $liquibaseServerPort --username $liquibaseUsername)\n        $liquibaseArguments += \"--password=`\"$liquibasePassword`\"\"\n\n\t\tbreak\n    }\n    \n    \"usernamepassword\"\n    {\n    \t# Add password\n        $liquibaseArguments += \"--password=`\"$liquibasePassword`\"\"\n        \n        break\n    }\n}\n\n# Add connection url\n$liquibaseArguments += \"--url=`\"$connectionUrl`\"\"\n\n\n\n# Determine if the output variable needs to be set\nif ($liquibaseCommand.EndsWith(\"SQL\"))\n{\n\t# Add the output variable as the command name\n    $liquibaseArguments += \"--outputFile=`\"$PSScriptRoot/artifacts/$($liquibaseCommand).sql`\"\"\n    \n    # Create the folder\n    if ((Test-Path -Path \"$PSScriptRoot/artifacts\") -eq $false)\n    {\n    \tNew-Item -Path \"$PSScriptRoot/artifacts\" -ItemType \"Directory\"\n    }\n}\n\n\n# Add the additional switches\nforeach ($liquibaseSwitch in $liquibaseAdditionalSwitches)\n{\n    $liquibaseArguments += $liquibaseSwitch\n}\n\nswitch ($liquibaseCommandStyle)\n{\n\t\"legacy\"\n    {\n    \t# Add the command to execute\n\t\t$liquibaseArguments += $liquibaseCommand\n    }\n    \"modern\"\n    {\n    \t# Insert the command at the beginning\n        $liquibaseArguments = @($liquibaseCommand) + $liquibaseArguments\n    }\n}\n\n# Add command arguments\n$liquibaseArguments += $liquibaseCommandArguments\n\n# Display what's going to be run\nif (![string]::IsNullOrWhitespace($liquibasePassword))\n{\n\t$liquibaseDisplayArguments = $liquibaseArguments.PSObject.Copy()\n    $arrayIndex = 0\n    for ($i = 0; $i -lt $liquibaseDisplayArguments.Count; $i++)\n    {\n    \tif ($null -ne $liquibaseDisplayArguments[$i])\n        {\n        \tif ($liquibaseDisplayArguments[$i].Contains($liquibasePassword))\n            {\n                $liquibaseDisplayArguments[$i] = $liquibaseDisplayArguments[$i].Replace($liquibasePassword, \"****\")\n        \t}\n        }\n    }\n    \n    Write-Host \"Executing the following command: $($liquibaseExecutable.FullName) $liquibaseDisplayArguments\"\n}\nelse\n{\n\tWrite-Host \"Executing the following command: $($liquibaseExecutable.FullName) $liquibaseArguments\"\n}\n\n\n# Redirection of stderr to stdout is done different on Windows versus Linux\nif ($IsWindows)\n{\n\t$liquibaseArguments += \"2>&1\"\n\t# Execute Liquibase\n\t& $liquibaseExecutable.FullName $liquibaseArguments\n}\n\nif ($IsLinux)\n{\n\t# Execute Liquibase\n\t& $liquibaseExecutable.FullName $liquibaseArguments 2>&1\n}\n\n# Check exit code\nif ($lastExitCode -ne 0)\n{\n\t# Fail the step\n    Write-Error \"Execution of Liquibase failed!\"\n}\n\n# Check to see if there were any files output\nif ((Test-Path -Path \"$PSScriptRoot/artifacts\") -eq $true)\n{\n\t# Loop through items\n    foreach ($item in (Get-ChildItem -Path \"$PSScriptRoot/artifacts\"))\n    {\n    \tNew-OctopusArtifact -Path $item.FullName -Name $item.Name\n    }\n}\n"
```

Comment:
4 x `# Check for Windows Authentcation type` becomes -> `# Check for Windows Authentication type`